map/routing_manager.hpp

```diff
@@ -65,14 +65,18 @@ class RoutingManager final
   {
     using IndexGetterFn = std::function<Index &()>;
     using CountryInfoGetterFn = std::function<storage::CountryInfoGetter &()>;
+    using CountryParentNameGetterFn = std::function<std::string(std::string const &)>;
 
-    Callbacks(IndexGetterFn && featureIndexGetter, CountryInfoGetterFn && countryInfoGetter)
+    Callbacks(IndexGetterFn && featureIndexGetter, CountryInfoGetterFn && countryInfoGetter,
```

Comment:
Is it possible to use perfect forwarding here?

Reply:
```c++
template <typename IndexGetter, CountryInfoGetter>
Callbacks(IndexGetter && indexGetter, CountryInfoGetter && countryInfoGetter)
: m_indexGetter(std::forward(indexGetter))
  ...
```
So, arguments are passed by universal reference here, the ctor is more flexible + perfect forwarding. WDYT?

Reply:
ok, done